deploy/vf-netns-switcher.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+
+netns=""
+pf=""
+conf_file=""
+pci=""
+
+TIMEOUT="10"
+POLL_INTERVAL="1"
+
+while test $# -gt 0; do
+  case "$1" in
+
+   --netns | -n)
+      netns=$2
+      shift
+      shift
+      ;;
+
+   --pf | -d)
+      pf=$2
+      shift
+      shift
+      ;;
+
+   --conf-file | -c)
+      conf_file=$2
+      shift
+      shift
+      ;;
+
+   --help | -h)
+      echo "
+vf-netns-switcher.sh --netns <> --pf <> [--conf-file <>]:
+
+	--netns | -n		Netns to switch the interface PFs and VFs to.
+
+	--pf | -d		The PF to switch it and its VFs to the specified netns.
+
+	--conf-file | -c	A file to read confs from, this will override cli flags.
+
+"
+      exit 0
+      ;;
+
+   *)
+      echo "No such option!!"
+      echo "Exitting ...."
+      exit 1
+  esac
+done
+
+get_pci_from_net_name(){
+    local interface_name=$1
+    local worker_netns="${2:-$netns}"
+
+    if [[ -z "$(ip l show $interface_name)" ]];then
+        if [[ -n "$(docker exec -t ${worker_netns} ip l show $interface_name)" ]];then
+             ip netns exec ${worker_netns} basename $(readlink /sys/class/net/${interface_name}/device)
+	     return 0
+        fi
+        echo ""
+        return 1
+    fi
+    basename $(readlink /sys/class/net/${interface_name}/device)
+}
+
+netns_create(){
+    local worker_netns="${1:-$netns}"
+
+    if [[ ! -e /var/run/netns/$worker_netns ]];then
+        local pid="$(docker inspect -f '{{.State.Pid}}' $worker_netns)"
+
+	if [[ -z "$pid" ]];then
```

Comment:
nit: in this line tab used instead of spaces for indentation

Reply:
thanks for the review and sorry for these small nits, i investigate if we can have some kind of CI that would do customized linting for bash.